toxav/bwcontroller.c

```diff
@@ -60,15 +61,17 @@ struct BWCMessage {
 static int bwc_handle_data(Messenger *m, uint32_t friendnumber, const uint8_t *data, uint16_t length, void *object);
 static void send_update(BWController *bwc);
 
-BWController *bwc_new(Messenger *m, uint32_t friendnumber, m_cb *mcb, void *mcb_user_data)
+BWController *bwc_new(Messenger *m, uint32_t friendnumber, m_cb *mcb, void *mcb_user_data,
+                      Mono_Time *toxav_given_mono_time)
```

Comment:
s/toxav_given_//

Reply:
why?

Reply:
lets do some other prefix, otherwise search for mono_time will find every mono_time

Reply:
done